dbt_schema_builder/app.py

```diff
@@ -42,26 +42,33 @@ def __init__(
         # Create a new, empty object to store a new version of our downstream
         # sources so we don't get any tables / models that may have been
         # deleted since the last run.
-        self.new_downstream_sources = self._generate_downstream_sources(database, no_pii)
+        self.new_downstream_sources = self._generate_downstream_sources(database, no_pii, pii_only)
 
-    def _generate_downstream_sources(self, database, no_pii=False):
+    def _generate_downstream_sources(self, database, no_pii=False, pii_only=False):
         """
         Generates the object to store the version of the downstream
         sources so we don't get any tables / models that may have been
         deleted since the last run. If no_pii flag is set we will exclude
         the downstream source name for that.
         """
-        ret_val = {
-            "version": 2,
-            "sources": [
-                {
-                    "name": self.safe_downstream_source_name,
-                    "database": database,
-                    "tables": [],
-                }
-            ],
-            "models": [],
-        }
+        if not pii_only:
```

Comment:
maybe take the approach as below and just append to the sources `if not pii_only` just to make this a bit cleaner and easier to follow. I know I was the one who started this trend so I accept this is me asking you to fix my mistake 😬

Reply:
Sounds good, rearranged